<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="WINAS251039-VKC" name="Test" tests="4" failures="1" timestamp="2 May 2017 09:40:51 GMT" time="24.567" errors="0">
  <testcase name="@AfterClass postCondition" time="0.0" classname="com.actitime.scripts.LoginTest">
    <failure type="java.lang.IllegalArgumentException" message="wrong number of arguments">
      <![CDATA[java.lang.IllegalArgumentException: wrong number of arguments
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- @AfterClass postCondition -->
  <testcase name="@BeforeClass preCondition" time="0.0" classname="com.actitime.scripts.CreateCustAndProjTest">
    <skipped/>
  </testcase> <!-- @BeforeClass preCondition -->
  <testcase name="@AfterClass postCondition" time="0.0" classname="com.actitime.scripts.CreateCustAndProjTest">
    <skipped/>
  </testcase> <!-- @AfterClass postCondition -->
  <testcase name="loginTest" time="18.612" classname="com.actitime.scripts.LoginTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
	at com.sun.proxy.$Proxy7.sendKeys(Unknown Source)
	at com.actitime.pom.LoginPage.login(LoginPage.java:24)
	at com.actitime.scripts.LoginTest.loginTest(LoginTest.java:22)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="createCustomerTest" time="0.001" classname="com.actitime.scripts.CreateCustAndProjTest">
    <skipped/>
  </testcase> <!-- createCustomerTest -->
  <testcase name="createProjectTest" time="0.001" classname="com.actitime.scripts.CreateCustAndProjTest">
    <skipped/>
  </testcase> <!-- createProjectTest -->
  <testcase name="deleteCustomerTest" time="0.001" classname="com.actitime.scripts.CreateCustAndProjTest">
    <skipped/>
  </testcase> <!-- deleteCustomerTest -->
</testsuite> <!-- Test -->
